exercises/nucleotide-count/tests/nucleotide-count.rs

```diff
@@ -9,10 +9,10 @@ fn check_dna(s: &str, pairs: &[(char, usize)]) {
     // check for the presence and value of each key in the given pairs vector.
     let mut m: HashMap<char, usize> = dna::nucleotide_counts(s).unwrap();
     for &(k, v) in pairs.iter() {
-        assert_eq!((k, m.remove(&k).unwrap()), (k, v));
+        assert_eq!((k, m.remove(&k)), (k, Some(v)));
     }
     // may fail with a message that clearly shows all extra pairs in the map
-    assert_eq!(m.iter().collect::<Vec<(&char,&usize)>>(), vec!());
+    assert_eq!(m.iter().collect::<Vec<(&char, &usize)>>(), vec!());
```

Comment:
the addition of a space is good, but isn't related to removing unwrap, so it should go in a separate commit